packages/client-core/src/admin/services/ScopeService.ts

```diff
@@ -27,6 +27,7 @@ store.receptors.push((action: ScopeActionType): any => {
         return s.merge({ fetching: true })
       case 'SCOPE_ADMIN_RETRIEVED':
         return s.merge({
+          //@ts-ignore
```

Comment:
use any rather than ts-ignore

Reply:
this is related to sequelize handling table name.